src/BitNode/BitNodeUtils.ts

```diff
@@ -9,3 +14,52 @@ export function isBitNodeFinished(): boolean {
   }
   return wd.backdoorInstalled;
 }
+
+export function canAccessBitNodeFeature(bitNode: number): boolean {
+  return Player.bitNodeN === bitNode || Player.activeSourceFileLvl(bitNode) > 0;
+}
+
+export function knowAboutBitverse(): boolean {
+  return Player.bitNodeOptions.activeSourceFiles.size > 0;
```

Comment:
I think this should be using the actual Player.sourceFiles

Reply:
I'm not sure I agree? You should be able to disable "knowing about the bitverse" (and the various features that enables) through some means, and turning off all SFs seems like the correct way.